examples/webgl_effects_displacement.html

```diff
@@ -0,0 +1,157 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<title>three.js webgl - effects - displacement</title>
+		<meta charset="utf-8">
+		<meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
+		<style>
+    body {
+      background: #000;
+      color: #fff;
+      padding: 0;
+      margin: 0;
+      overflow: hidden;
+      font-family: georgia;
+      text-align: center;
+    }
+    a {
+      color: skyblue;
+      text-decoration: none
+    }
+    #info {
+      position: absolute;
+      width: 100%;
+      top: 10px;
+    }
+		</style>
+	</head>
+
+	<body>
+    <div id="info">
+        Depth Displacement Effect<br/> Written
+        by <a href="http://orfleisher.com">juniorxsound</a>
+    </div>
+
+		<script src="../build/three.js"></script>
+
+		<script src="js/effects/DepthDisplacement.js"></script>
+
+		<script src="js/Detector.js"></script>
+		<script src="js/libs/stats.min.js"></script>
+
+		<script>
+
+			if ( ! Detector.webgl ) Detector.addGetWebGLMessage();
+
+			var container, stats;
+
+			var camera, scene, renderer, effect;
+
+      init();
```

Comment:
Your code needs some cleanup seems it looks like you mix hard with soft tabs. There are also some code style issues. Please have a look at: https://github.com/mrdoob/three.js/wiki/Mr.doob%27s-Code-Style%E2%84%A2

Reply:
Thanks, followed the guidelines and committed changes

Reply:
Much better! 👍